packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1850,6 +1850,156 @@ describe('postgresql introspection stopgaps', () => {
+          // *** WARNING ***
+          // 
+          // These objects have comments defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/database-comments
+          // - Type "model", Name "a"
+          // - Type "field", Name "a.val"
+          // 
+        `)
+        expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+        expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      })
+    })
+
+    describe('36/2 - comments found: views & fields', () => {
+      const variant = 2
+      setupPostgresForWarning(stopGap, warningCode, variant)
+
+      test('basic introspection', async () => {
+        ctx.fixture(`introspection/postgresql/${stopGap}-warning-${warningCode}-${variant}`)
+        const introspect = new DbPull()
+        const result = introspect.parse(['--print'])
+
+        await expect(result).resolves.toMatchInlineSnapshot(``)
+
+        expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+          generator client {
+            provider        = "prisma-client-js"
+            previewFeatures = ["views"]
+          }
+
+          datasource db {
+            provider = "postgres"
+            url      = env("TEST_POSTGRES_URI_MIGRATE")
+          }
+
+          model a {
+            id  Int     @id
+            val String? @db.VarChar(20)
+          }
+
+          /// The underlying view does not contain a valid unique identifier and can therefore currently not be handled by the Prisma Client.
+          /// This view or at least one of its fields has comments in the database, and requires an additional setup for migrations: Read more: https://pris.ly/d/database-comments
+          view b {
+            val String? @db.VarChar(20)
+
+            @@ignore
+          }
+
+
+          // introspectionSchemaVersion: NonPrisma,
+        `)
+        expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+        expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+
+          // *** WARNING ***
+          // 
+          // The following views were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client. Please refer to the documentation on defining unique identifiers in views: https://pris.ly/d/view-identifiers
+          // - View "b"
+          // 
+          // These objects have comments defined in the database, which is not yet fully supported. Read more: https://pris.ly/d/database-comments
+          // - Type "view", Name "b"
+          // - Type "field", Name "b.val"
+          // 
+        `)
+        expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+        expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+      })
+    })
+
+    describe('36/3 - comments found: types', () => {
```

Comment:
enums only, right? dont think we support other types
```suggestion
    describe('36/3 - comments found: enums', () => {
```

Reply:
Yep, this is just enums (in engines) for now, can change this to be less generic if that's what we want